### PR TITLE
Style/improve network selector dropdown

### PR DESCRIPTION
**Pull request by antonyayoub**
# Name of the feature/issue
Improve Network selector dropdown

## Description
- Allowing scroll on the Network selector drop-down on the main screen. 
- Add CTA at the bottom that opens the Networks view. 

## Type of change

Click the correct/s option/s

- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  New feature (non-breaking change which adds functionality)
- [ ]  Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ]  This change requires a documentation update

### CODE REVIEW

**Review by fpezzola — APPROVED** [review]
lgtm!

File: packages/ui/src/components/input/NetworkSelect.tsx [hunk 1]
```diff
@@ -159,6 +162,17 @@ const NetworkSelect: FunctionComponent<{
                                     }
                                 />
                             ))}
+                    <li
```
Comment (julianariel) [inline packages/ui/src/components/input/NetworkSelect.tsx:165]: we have a `LinkButton` component that can probably be used here

Comment (antonyayoub) [reply packages/ui/src/components/input/NetworkSelect.tsx:165]: I tried using it. It is mainly made for buttons like "Next", "Done", "Cancel" and so on. 
To use it in this context, I must override all the CSS using the classes prop.

Comment (julianariel) [reply packages/ui/src/components/input/NetworkSelect.tsx:165]: sorry I made a mistake on the intended component, the correct one would be `ClickableText`

**Review by julianariel — APPROVED** [review]
LGTM!

**Pull request merged**